Makefile

```diff
@@ -63,7 +63,7 @@ ldflags := $(strip $(ldflags))
 BUILD_FLAGS := -tags $(build_tags_comma_sep) -ldflags '$(ldflags)' -trimpath
 
 # The below include contains the tools target.
-include contrib/devtools/Makefile
+#include contrib/devtools/Makefile
```

Comment:
may need to also drop related target dependencies from here, like this one https://github.com/fetchai/fetchd/pull/99/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R135

the linting isn't passing anyway, we can probably get ride of it for now and reintroduce a bit later - just to make the all target pass (now only make install works)

Reply:
Yes make sense. I will do a run through the rest of the make targets and have a bit of a tidy up here